config/application.yml.default

```diff
@@ -24,6 +24,8 @@ aamva_verification_request_timeout: 5.0
 aamva_verification_url: https://example.org:12345/verification/url
 account_creation_device_profiling: disabled
 account_reset_fraud_user_wait_period_days:
+account_reset_request_attempt_window_in_days: 2
+account_reset_request_max_attempts: 2
 account_reset_token_valid_for_days: 1
```

Comment:
I don't have strong opinions on what the values here should be, but it should be monitored for potential adjustments